package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "mattermost-mobile",
-      "version": "1.50.0",
```

Comment:
Are changes in packate-lock.json expected?

Reply:
not expected but the change is right